crates/telio-model/src/api_config.rs

```diff
@@ -244,19 +249,27 @@ pub struct FeatureDerp {
 }
 
 /// Whether to validate keys
-#[derive(Clone, Debug, PartialEq, Eq, Deserialize)]
-pub struct FeatureValidateKeys(pub bool);
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Deserialize)]
+pub struct FeatureValidateKeys {
+    /// Defaults to true
+    pub validate_keys: bool,
+}
 
 impl Default for FeatureValidateKeys {
     fn default() -> Self {
-        Self(true)
+        Self {
+            validate_keys: true,
+        }
     }
 }
 
 /// Turns on connection resets upon VPN server change
 #[derive(Default, Clone, Copy, Debug, PartialEq, Eq, Deserialize)]
 #[serde(transparent)]
-pub struct FeatureBoringtunResetConns(pub bool);
+pub struct FeatureBoringtunResetConns {
+    /// Defults to false
+    pub reset_connections: bool,
+}
```

Comment:
Reminder for myself: this change will require a change in the confluence page for libtelio features

Reply:
why is that? It seems that the json format is the same 🤔

Reply:
Yes, I updated these to not have to do custom conversion logic for UniFFI but of course it's better to keep the API simple than to avoid such conversions. Fixed